artemis.jh

```diff
@@ -302,6 +308,29 @@ entity ApollonDiagram {
     jsonRepresentation String       // LONGTEXT --> clob
 }
 
+entity Complaint {
+    text String,
+    accepted Boolean,
```

Comment:
What does accepted mean?

Reply:
A complaint is marked as accepted if it is considered valid, so it has brought to some review of the assessment, therefore it doesn't count against the limit